release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -447,6 +447,10 @@ Updated boot images has been promoted to GA for Amazon Web Services (AWS) cluste
 [id="ocp-release-notes-networking_{context}"]
 === Networking
 
+[id="ocp-release-notes-networking-whitelist-blacklist-annotation-updated-allowlist-denylist_{context}"]
+=== Updated terminology for "whitelist" and "blacklist" annotations
+The terminology for the `ip_whitelist` and `ip_blacklist` annotations have been updated to `ip_allowlist` and `ip_denylist`, respectively. Currently, {product-name} still supports the `ip_whitelist` and `ip_blacklist` annotations. However, these annotations will be removed in a future release.
```

Comment:
```suggestion
The terminology for the `ip_whitelist` and `ip_blacklist` annotations have been updated to `ip_allowlist` and `ip_denylist`, respectively. Currently, {product-name} still supports the `ip_whitelist` and `ip_blacklist` annotations. However, these annotations are planned for removal in a future release.
```

Reply:
I've always gotten the feedback to say "planned" instead of "will be" when we promise things in certain releases. I certainly hope these annotations will be removed, but just thought I'd comment my feedback anyways.

Reply:
https://redhat-documentation.github.io/supplementary-style-guide/#statements-about-the-future

Reply:
That's awesome. Thanks for letting me know.